libs/shared/form-fields/src/lib/RadioController/RadioController.tsx

```diff
@@ -58,10 +78,32 @@ export const RadioController: FC<Props> = ({
                 disabled={disabled}
                 errorMessage={index === options.length - 1 ? error : undefined}
                 hasError={error !== undefined}
-              />
-            ))}
-          </Stack>
-        )
+              />,
+            ),
+          )
+
+        if (halve) {
+          return (
+            <Box
+              display="flex"
+              flexWrap="wrap"
+              flexDirection="row"
+              className={cs(negativeMarginTop, negativeMarginLeft)}
+            >
+              {renderRadioButton((child) => (
+                <Box
+                  paddingLeft={leftSpace}
+                  paddingTop={topSpace}
+                  className={styles.coverHalf}
+                >
+                  {child}
+                </Box>
+              ))}
+            </Box>
```

Comment:
is there any reason why we are using custom layout here rather than using `GridRow` and `GridColumn`?

Reply:
Ahh nice, that would also solve allowing further splitting :+1: